Makefile

```diff
@@ -10,6 +10,7 @@ clean:
 
 format:
 	black tests
+	grep -lPR '\t' patterns/ | xargs sed -i 's/\t/  /g'
```

Comment:
This will fail when nothing matches. `make format` isn't part of `make test` or anything so just a nit!

```
grep -lPR '\t' patterns/ | xargs sed -i 's/\t/  /g'
sed: no input files
make: *** [Makefile:13: format] Error 123
```

Reply:
Fixed it by setting `xargs -r` so it won't run sed when there are no files :+1: Good call out.